internal/backends/testing/Cargo.toml

```diff
@@ -21,10 +22,18 @@ path = "lib.rs"
 internal = []
 # ffi for C++ bindings
 ffi = []
+system-testing = ["quick-protobuf", "pb-rs", "generational-arena", "smol", "byteorder"]
 
 [dependencies]
 i-slint-core = { workspace = true }
 vtable = { workspace = true }
+quick-protobuf = { version = "0.8.1", optional = true }
+generational-arena = { version = "0.2.9", optional = true }
+smol = { version = "2.0.0", optional = true }
```

Comment:
I wonder if it is not better to use `async-net` crate instead of smol

Reply:
Excellent idea, one indirection less.